### PR TITLE
Update accessibility styling on topic pages

### LINKED ISSUE #9970: Add accessible behaviour to Topic pages
There are three accessibility requirements missing from the new Topic pages:

- The 'Skip to content' link should anchor to the h1 when used
- Add specific colour to h1 element as specified in the design (e.g. do not inherit a colour)
- Focus state of promo links should include an underline as well as the browser default outline (currently it uses browser default only)
- Add visited link styling to links in promos

**To Reproduce**
*Skip to content h1*
Steps to reproduce the behaviour:

1. Go to 'https://www.test.bbc.com/pidgin/new_topics/cqywjyzk2vyt'
2. Tab to the 'Skip to content' link and press enter
3. Page stays at top 

**Expected behaviour**
Page should move down to anchor where the h1 begins (example can be seen here: https://www.bbc.com/pidgin)

*Specific colour to h1 element* (Firefox example)
Steps to reproduce the behaviour:
1. Go to 'https://www.test.bbc.com/pidgin/new_topics/cqywjyzk2vyt'
2. Change colours to custom colours and enable
3. Disable custom colours
4. H1 element is still custom colour

**Expected behaviour**
H1 text colour should change back to default colour

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves #9970

**Overall change:**
- Add header colour to ensure it isn't overwritten when using colour settings on Firefox
- Skip to content link now anchors to h1
- Visited links now have a different colour
- Focused state now includes an underline

**Code changes:**

- Add tab index to h1
- Add id of 'content' to allow skip to content to move to header
- Add h1 styling to topic header component
- Add visited and focus states to Topic link component

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/TopicPage/TopicTitle/index.jsx [hunk 1]
```diff
@@ -17,7 +17,8 @@ import {
 
 const H1 = styled.h1`
   ${({ service }) => getSansBold(service)}
-  margin: ${GEL_SPACING_TRPL}rem 0 ${GEL_SPACING_TRPL}rem 0;
+  color: #141414;
```
Comment (greenc05) [inline src/app/pages/TopicPage/TopicTitle/index.jsx]: Should this value be hard-coded?

Comment (nicola-d-bbc) [reply src/app/pages/TopicPage/TopicTitle/index.jsx]: Nope, had to add to Psammead first.

File: src/app/components/Promo/a.jsx [hunk 2]
```diff
@@ -1,12 +1,18 @@
 import styled from '@emotion/styled';
-import { C_EBON } from '@bbc/psammead-styles/colours';
+import { C_GREY_6, C_GREY_10 } from '@bbc/psammead-styles/colours';
 
 const A = styled.a`
-  color: ${C_EBON};
+  color: ${C_GREY_10};
   text-decoration: none;
   &:hover {
```
Comment (jroebu14) [inline src/app/components/Promo/a.jsx]: no biggie but I think you can do

```suggestion
  &:hover, &:focus {
```

and remove the &:focus style below

File: src/app/pages/TopicPage/TopicTitle/index.jsx [hunk 3]
```diff
@@ -40,7 +42,7 @@ const H1 = styled.h1`
 const TopicTitle = ({ children }) => {
   const { script, service } = useContext(ServiceContext);
   return (
-    <H1 service={service} script={script}>
+    <H1 service={service} script={script} id="content" tabindex="-1">
```
Comment (jroebu14) [inline src/app/pages/TopicPage/TopicTitle/index.jsx]: does the tabindex not need to be `tabIndex`?

File: src/app/pages/TopicPage/TopicTitle/index.jsx [hunk 4]
```diff
@@ -17,7 +18,8 @@ import {
 
 const H1 = styled.h1`
   ${({ service }) => getSansBold(service)}
-  margin: ${GEL_SPACING_TRPL}rem 0 ${GEL_SPACING_TRPL}rem 0;
+  color: ${C_GREY_10};
+  margin: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING_TRPL} 0;
```
Comment (jroebu14) [inline src/app/pages/TopicPage/TopicTitle/index.jsx]: you can use shorthand margin/padding here

```suggestion
  margin: ${GEL_SPACING_TRPL} 0;
```

**Review by jroebu14 — APPROVED** [review]
Couple of minor improvements you can make otherwise looks good 👍

**Review by amoore108 — APPROVED** [review]

**Pull request merged**